app/src/main/java/com/github/capntrips/kernelflasher/ui/screens/slot/SlotViewModel.kt

```diff
@@ -89,7 +92,12 @@ class SlotViewModel(
     }
 
     fun refresh(context: Context) {
-        Shell.cmd("/data/adb/magisk/magiskboot unpack $boot").exec()
+        // init magiskboot
+        if (!File(STOCK_MAGISKBOOT).exists()) {
+            magiskboot = context.applicationInfo.nativeLibraryDir + "/libmagiskboot.so"
```

Comment:
It almost certainly isn't the best way to handle it, but let's keep it consistent with the other binaries and drop it in the [assets](https://github.com/capntrips/KernelFlasher/tree/v1.0.0-alpha012/app/src/main/assets) folder and copy it to the files folder [here](https://github.com/capntrips/KernelFlasher/blob/v1.0.0-alpha012/app/src/main/java/com/github/capntrips/kernelflasher/MainActivity.kt#L154-L155). There's no need to include the x86 binaries, as these architectures aren't currently supported, or the arm64 binary, as the arm one should be sufficient for both.

Reply:
I will do that

Reply:
To clarify, lets keep the arm32 binary, rather than the arm64 one, to maintain compatibility with arm32 devices. If support is eventually dropped for arm32 binaries, I'll have to revisit this for lptools and Hashtree Patcher, as well.